code/modules/mob/living/carbon/human/human.dm

```diff
@@ -30,8 +30,6 @@
 
 	create_reagents(330)
 
-	martial_art = GLOB.default_martial_art
```

Comment:
Only difference from the TG PR is here.
https://github.com/tgstation/tgstation/pull/27324/files#diff-1ff8328c73a4a8a73ad2c1fe6d517588R39-R40
This will not trigger on our side and having it null is not an issue since it's checked before anyway. It only caused some useless checks to be done every punch etc.